packages/appium/lib/cli/parser.js

```diff
@@ -280,6 +284,35 @@ class ArgParser {
       }
     }
   }
+
+  /**
+   *
+   * @param {import('argparse').SubParser} subParser
+   */
+    static _addSetupToParser(subParser) {
+      const setupParser = subParser.add_parser('setup', {
+        add_help: true,
+        help: 'Install latest uiautomator2 and xcuitest driver if no APPIUM_HOME was empty',
```

Comment:
what does `if no APPIUM_HOME was empty` mean? Do you mean to say `unless APPIUM_HOME isn't empty`, of `if APPIUM_HOME is empty`?

Reply:
yea, that was my bad. Right now is:

```
Install latest preset drivers and plugins if APPIUM_HOME has no drivers and plugins
```